src/services/EcoBridge/Arbitrum/ArbitrumBridge.lists.ts

```diff
@@ -0,0 +1,29 @@
+// NOTE: List used by official Arbitrum Bridge
+
+export const ARBITRUM_TOKEN_LISTS_CONFIG = [
```

Comment:
I usually don't like to have my code and my data mixed together.
Can we have this in a json file and read it? Or even something else.

Reply:
Resolved by #900